<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>70</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="VSpacerMain">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="HLayoutButtons">
      <item>
       <widget class="QPushButton" name="ButtonHelp">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Help!</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="HSpacerButtons">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonRefresh">
        <property name="text">
         <string>Refresh</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/pic/USBstick.png</normaloff>:/pic/USBstick.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonPrevious">
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/pic/LeftArrow.png</normaloff>:/pic/LeftArrow.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonNext">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/pic/RightArrow.png</normaloff>:/pic/RightArrow.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
